lib/resources/r4/device.yaml

```
@@ -12,3 +12,15 @@ fields:
       - display: SNOMED CT
         system: http://snomed.info/sct
         info_link: https://www.hl7.org/fhir/valueset-device-type.html
+
+- name: deviceName.type
+  required: 'No'
+  type: code
+  description: The type of deviceName.
+  binding:
+    description: The type of name the device is referred by.
+    terminology:
+      - display: User Friendly name
+        info_link: https://hl7.org/fhir/r4/codesystem-device-nametype.html#device-nametype-user-friendly-name
+      - display: Manufacturer name
+        info_link: https://hl7.org/fhir/r4/codesystem-device-nametype.html#device-nametype-manufacturer-name
```

Not quite right yet. The different items in the "terminology" list are principally used to document the different value systems which are used to compose a ValueSet rather than for listing individual values within the ValueSet.

In this case I was anticipating an using the description line to detail the allowed codes as was done in the ChargeItem link in my previous comment

Something like this (which I haven't tested locally):
```suggestion
- name: deviceName.type
  required: 'No'
  type: code
  description: The type of deviceName.
  binding:
    description: The type of deviceName. <code>Device Name Type</code> can be either <code>user-friendly-name</code> or <code>manufacturer-name</code>.
    terminology:
      - display: Device Name Type
        system: http://hl7.org/fhir/device-nametype
        info_link: https://hl7.org/fhir/R4/valueset-device-nametype.html
```

---

Oh, yeah. Now I understand. I will commit the changes